ctapipe/utils/unstructured_interpolator.py

```diff
@@ -28,23 +28,25 @@ class UnstructuredInterpolator:
     In the case that a numpy array is passed as the interpolation values this class will
     behave exactly the same as the scipy LinearNDInterpolator
     """
-    def __init__(self, interpolation_points, function_name=None, remember_last=False,
+    def __init__(self, keys, values, function_name=None, remember_last=False,
                  bounds=None, dtype=None):
         """
         Parameters
         ----------
-        interpolation_points: dict
-            Dictionary of interpolation points (stored as key) and values
+        keys: ndarray
+            Interpolation grid points
+        values: ndarray
+            Interpolation values
         function_name: str
             Name of class member function to call in the case we are interpolating
             between class predictions, for numpy arrays leave blank
         """
-
-        self.keys = np.array(list(interpolation_points.keys()))
+        print("bounds",bounds)
```

Comment:
use the Component logging system here: `self.log.debug("bounds=%s", bounds)` , not `print()`

Reply:
This print statement was accidentally left in anyway. I'll just remove it.